scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
Ah, so is this how it goes then:

1. We copy html files and scripts to /templates/prod/head for serving the html files
2. Scripts are copied after that to build/ for minification and serving

This is a bit confusing to me. I know the html files are not being directly served, but why do we need /templates/prod/head if all of its contents are being copied to /build?